mypy_django_plugin/transformers/managers.py

```diff
@@ -71,25 +72,37 @@ def get_method_type_from_dynamic_manager(
     queryset_info = helpers.lookup_fully_qualified_typeinfo(api, queryset_fullname)
     assert queryset_info is not None
 
-    def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[ProperType]:
-        # TODO: Handle @overload?
-        if isinstance(definition, FuncBase) and not isinstance(definition, OverloadedFuncDef):
-            return definition.type
-        elif isinstance(definition, Decorator):
-            return definition.func.type
-        return None
+    is_fallback_queryset = queryset_info.metadata.get("django", {}).get("any_fallback_queryset", False)
 
-    method_type = get_funcdef_type(queryset_info.get_method(method_name))
-    if method_type is None:
-        return None
+    method_type = _get_funcdef_type(queryset_info.get_method(method_name))
+    if method_type is None or not isinstance(method_type, FunctionLike):
```

Comment:
I'm always split about this. This should work, but is it reasonable?

```suggestion
    if not isinstance(method_type, FunctionLike):
```